jps-plugin/src/org/intellij/erlang/jps/builder/ErlangBuilder.java

```diff
@@ -83,19 +80,23 @@ public void build(@NotNull ErlangTarget target,
                     @NotNull DirtyFilesHolder<ErlangSourceRootDescriptor, ErlangTarget> holder,
                     @NotNull BuildOutputConsumer outputConsumer,
                     @NotNull CompileContext context) throws ProjectBuildException, IOException {
-    LOG.debug(target.getPresentableName());
-    if (!holder.hasDirtyFiles() && !holder.hasRemovedFiles()) return;
 
     JpsModule module = target.getModule();
     JpsProject project = module.getProject();
     ErlangCompilerOptions compilerOptions = JpsErlangCompilerOptionsExtension.getOrCreateExtension(project).getOptions();
     if (compilerOptions.myUseRebarCompiler) return;
 
+    LOG.info("Starting build for " + target.getPresentableName());
```

Comment:
Try to make log messages more concise and informative.

Here is your log output sample:
```
2015-08-20 19:43:21,611 [   1204]   INFO - lang.jps.builder.ErlangBuilder - Starting prepare dependencies
2015-08-20 19:43:21,682 [   1275]   INFO - lang.jps.builder.ErlangBuilder - Starting build for Erlang 'empty'
2015-08-20 19:43:21,683 [   1276]   INFO - lang.jps.builder.ErlangBuilder - Compile source code for module "empty".
2015-08-20 19:43:22,168 [   1761]   INFO - lang.jps.builder.ErlangBuilder - Compile tests for module "empty".
```

It could look something like this instead:
```
2015-08-20 19:43:21,611 [   1204]   INFO - lang.jps.builder.ErlangBuilder - Computing dirty files
2015-08-20 19:43:21,682 [   1275]   INFO - lang.jps.builder.ErlangBuilder - Building module 'empty'
2015-08-20 19:43:21,683 [   1276]   INFO - lang.jps.builder.ErlangBuilder - Building production
2015-08-20 19:43:22,168 [   1761]   INFO - lang.jps.builder.ErlangBuilder - Building tests
```

Also, I'd consider adding more information to logs - these particular messages do not provide any useful information in case something goes wrong.